lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -138,27 +137,24 @@ sub list_running_ajax {
         ],
     );
 
-    my @running;
-    while (my $job = $running->next) {
-        my $job_id = $job->id;
-        push(
-            @running,
-            {
-                DT_RowId => 'job_' . $job_id,
-                id => $job_id,
-                clone => $job->clone_id,
-                test => $job->TEST . '@' . ($job->MACHINE // ''),
-                distri => $job->DISTRI // '',
-                version => $job->VERSION // '',
-                flavor => $job->FLAVOR // '',
-                arch => $job->ARCH // '',
-                build => $job->BUILD // '',
-                testtime => ($job->t_started // '') . 'Z',
-                group => $job->group_id,
-                state => $job->state,
-                progress => $job->progress_info,
-            });
-    }
+    my @running = map {
```

Comment:
Hm, I remember this change from a previous PR, and I think I already wrote that I don't see the advantage here.
`->next` might actually be preferrable to `->all` which reads all DBIC objects into memory